keps/sig-network/3070-reserved-service-ip-range/README.md

```diff
@@ -0,0 +1,444 @@
+# KEP-3070: Reserve Service IP Range For Dynamic and Static IP Allocation
```

Comment:
either "Range for Static ..." or "Range**s** for Dynamic and Static"

Reply:
the later :+1: